packages/dds/tree/src/test/sequence-change-family/randomSequenceGenerator.spec.ts

```diff
@@ -0,0 +1,65 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { makeRandom } from "@fluid-internal/stochastic-test-utils";
+import { ChangeRebaser } from "../../rebase";
+import { AnchorSet, FieldKey } from "../../tree";
+import { brand, changeCombinator, generateRandomChange, generateRandomUpPaths } from "../../util";
+
+const testSeed = 432167897;
+
+function commutativeRebaser<TChange>(data: {
+    compose: (changes: TChange[]) => TChange;
+    invert: (changes: TChange) => TChange;
+    rebaseAnchors: (anchor: AnchorSet, over: TChange) => void;
+}): ChangeRebaser<TChange> {
+    return {
+        rebase: (change: TChange, over: TChange) => change,
+        ...data,
+    };
+}
+
+const counterRebaser = commutativeRebaser({
+    compose: (changes: number[]) => changes.reduce((a, b) => a + b, 0),
+    invert: (change: number) => -change,
+    rebaseAnchors: (anchor: AnchorSet, over: number) => {},
+});
```

Comment:
Since Craig committed his PR you may be able to import these instead of redefining them here.